server/database/requests/tests/conversationToMessage.test.js

```diff
@@ -0,0 +1,27 @@
+let conversationToMessageClass = require('../conversationToMessages');
+const chai = require("chai");
+const expect = chai.expect;
+
+describe("ConversationToMessages", function() {
+    let conversationToMessage;
+    before(() => {
+        conversationToMessage = new conversationToMessageClass();
+    });
+
+    beforeEach(async (done) => {
+        await conversationToMessage.clear();
+        done();
+    });
+
+    it("should create conversation", async (done) => {
+        await conversationToMessage.create("1223");
+        expect(await conversationToMessage.get("1223")).to.be.an("Array");
+        done();
+    });
```

Comment:
Look on my general note about returning in functions. 
In addition you are using the original get function. means that if the get function is bugged this test will fail. means it's not UT any more. you need to isolate your function. access to the _data property to check if it really set it.